<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemSettingsDlg</class>
 <widget class="QDialog" name="SystemSettingsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SystemSettingsDlg</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#wgt_title{
    background-color: rgb(0, 144, 216);
}
#wgt_main{
	background-color: #eeeeee;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="wgt_title" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#lbl_title{
font-family:&quot;微软雅黑&quot;;
font-size:30px;
color:white;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lbl_title">
        <property name="text">
         <string>系统配置</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wgt_main" native="true">
     <property name="styleSheet">
      <string notr="true">QLabel,QLineEdit,QPushButton{
font-family:&quot;微软雅黑&quot;;
font-size:16px;
}
#wgt1,#wgt2,#wgt3,#wgt4,#wgt5,#wgt6,#wgt7{
border:1px solid gray;
background:#ffffff;
}
#wgt1,#wgt2,#wgt3,#wgt4,#wgt5,#wgt6{
border-bottom:none;
}
#wgt1,#wgt3{
border-right:none;
}
#btn_cancel,#btn_commit{
color:#ffffff;
background:#00cb00;
border:none;
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="topMargin">
       <number>120</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>60</number>
      </property>
      <item row="0" column="1" rowspan="2" colspan="2">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <property name="spacing">
           <number>20</number>
          </property>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QWidget" name="wgt1" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>51</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#lbl_dev{
background-color: #d4d4d4;
padding-right:6px;
border:1px solid gray;
border-right:none;
border-bottom:none;
}
#lbl_devNo{
padding-left:6px;
}</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="lbl_dev">
                 <property name="minimumSize">
                  <size>
                   <width>134</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>设备编号</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lbl_devNo">
                 <property name="minimumSize">
                  <size>
                   <width>268</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QWidget" name="wgt2" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>51</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#lbl_ver{
background-color: #d4d4d4;
padding-right:6px;
border:1px solid gray;
border-right:none;
border-bottom:none;
}
#lbl_verNo{
padding-left:6px;
}</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="lbl_ver">
                 <property name="minimumSize">
                  <size>
                   <width>134</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>版本号</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lbl_verNo">
                 <property name="minimumSize">
                  <size>
                   <width>268</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QWidget" name="wgt3" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>51</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#lbl_time{
background-color: #d4d4d4;
padding-right:6px;
border:1px solid gray;
border-right:none;
border-bottom:none;
}
#lbl_timeNo{
padding-left:6px;
}</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="lbl_time">
                 <property name="text">
                  <string>运行时间</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lbl_timeNo">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QWidget" name="wgt4" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>51</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#lbl_type{
background-color: #d4d4d4;
padding-right:6px;
border:1px solid gray;
border-right:none;
border-bottom:none;
}
#lbl_typeNo{
padding-left:6px;
}</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="lbl_type">
                 <property name="minimumSize">
                  <size>
                   <width>134</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>摄像头类型</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lbl_typeNo">
                 <property name="text">
                  <string>双目摄像头/广角摄像头</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QWidget" name="wgt5" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>51</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#lbl_ser1{
background-color: #d4d4d4;
padding-right:6px;
border:1px solid gray;
border-right:none;
border-bottom:none;
}
#lineEdit_ser1{
padding-left:6px;
}</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,0,0">
               <property name="spacing">
                <number>12</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>12</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="lbl_ser1">
                 <property name="minimumSize">
                  <size>
                   <width>134</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>134</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>服务器地址</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lbl_ser">
                 <property name="minimumSize">
                  <size>
                   <width>500</width>
                   <height>37</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>500</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
            <item row="3" column="0" colspan="2">
             <widget class="QWidget" name="wgt6" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>51</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#lbl_ser2{
background-color: #d4d4d4;
padding-right:6px;
border:1px solid gray;
border-right:none;
border-bottom:none;
}
#lineEdit_ser2{
padding-left:6px;
}</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,5,0">
               <property name="spacing">
                <number>12</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>12</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="lbl_ser2">
                 <property name="minimumSize">
                  <size>
                   <width>134</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>134</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>修改服务器地址</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_ser">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>37</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>500</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
            <item row="4" column="0" colspan="2">
             <widget class="QWidget" name="wgt7" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>51</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#lbl_devIn{
background-color: #d4d4d4;
padding-right:6px;
border:1px solid gray;
border-right:none;
}
#lineEdit_devIn{
padding-left:6px;
}</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,5,0">
               <property name="spacing">
                <number>12</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>12</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="lbl_devIn">
                 <property name="minimumSize">
                  <size>
                   <width>134</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>134</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>设备编号</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_devIn">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>37</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>500</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>12</number>
            </property>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btn_exit">
              <property name="minimumSize">
               <size>
                <width>48</width>
                <height>48</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_min">
              <property name="minimumSize">
               <size>
                <width>48</width>
                <height>48</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_reboot">
              <property name="minimumSize">
               <size>
                <width>48</width>
                <height>48</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_shutdown">
              <property name="minimumSize">
               <size>
                <width>48</width>
                <height>48</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <property name="spacing">
           <number>20</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_cancel">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>51</height>
             </size>
            </property>
            <property name="text">
             <string>取消</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_commit">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>51</height>
             </size>
            </property>
            <property name="text">
             <string>提交</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="3">
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>217</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>217</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AIOViewer.qrc"/>
 </resources>
 <connections/>
</ui>
